TabloidMVC/Controllers/PostController.cs

```diff
@@ -20,9 +24,22 @@ public PostController(IPostRepository postRepository, ICategoryRepository catego
             _categoryRepository = categoryRepository;
         }
 
+        //Posts sorted by PublishDateTime newest first
         public IActionResult Index()
         {
+            //Return posts
             var posts = _postRepository.GetAllPublishedPosts();
+            //Sort by PublishDateTime
+            posts.Sort((y, x) => DateTime.Compare((DateTime)x.PublishDateTime, (DateTime)y.PublishDateTime));
```

Comment:
I don't think you need this line now that you're sorting the posts in the sql command.